mux.go

```diff
@@ -0,0 +1,215 @@
+package e5e
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/signal"
+	"runtime"
+	"strings"
+)
+
+// mux defines a container for entrypoints and routes the requests for the given entrypoint
+// to the dedicated handlers.
+type mux struct {
+	handlers    map[string]HandlerFactory
+	stdinReader *bufio.Scanner
+}
+
+var globalMux = &mux{handlers: make(map[string]HandlerFactory)}
+
+func Start(ctx context.Context) {
```

Comment:
The error handling is not consistent. Sometimes `go-e5e` is prepended for errors / panics (which should be the case). Sometimes it's not.

Reply:
This is by intent, as all panics (except the one by the reader goroutine) are done by the `Start` method. In my opinion, only the last error needs to be prefixed with the `go-e5e: ` prefix, not all of the nested errors as well.